gcloud/logging/_gax.py

```diff
@@ -120,7 +120,12 @@ def logger_delete(self, project, logger_name):
         """
         options = None
         path = 'projects/%s/logs/%s' % (project, logger_name)
-        self._gax_api.delete_log(path, options)
+        try:
+            self._gax_api.delete_log(path, options)
+        except GaxError as exc:
+            if exc_to_code(exc.cause) == StatusCode.NOT_FOUND:
+                raise NotFound(path)
```

Comment:
I based it on

```
GaxError: GaxError(RPC failed, caused by AbortionError(code=StatusCode.NOT_FOUND, details="Requested entity was not found."))
```

Which gives

```
Requested entity was not found.
```

as the details (yes it's in `exc.cause.details`)